libs/sdk-ui-filters/src/AttributeFilter/hooks/useResolveParentFiltersInput.ts

```diff
@@ -12,12 +12,27 @@ import { ParentFilterOverAttributeType } from "../types.js";
 export const useResolveParentFiltersInput = (
     parentFilters?: AttributeFiltersOrPlaceholders,
     overAttribute?: ParentFilterOverAttributeType,
-) => {
+    supportsSettingConnectingAttributes?: boolean,
+): IElementsQueryAttributeFilter[] => {
     const resolvedParentFilters = useResolveValueWithPlaceholders(parentFilters);
 
     return useMemo(() => {
+        if (!supportsSettingConnectingAttributes) {
+            return getParentFiltersWithoutOverAttribute(resolvedParentFilters);
+        }
+
         return getParentFiltersWithOverAttribute(resolvedParentFilters, overAttribute);
-    }, [resolvedParentFilters, overAttribute]);
+    }, [supportsSettingConnectingAttributes, resolvedParentFilters, overAttribute]);
+};
+
+const getParentFiltersWithoutOverAttribute = (
+    parentFilters: IAttributeFilter[],
+): IElementsQueryAttributeFilter[] => {
+    if (!parentFilters) {
+        return [];
+    }
+
+    return parentFilters.map((attributeFilter) => ({ attributeFilter, overAttribute: null }));
```

Comment:
`overAttribute: null` looks suspicious - is it just because of the typings?

Reply:
Yes, this is just for the typing so that we would not need to break the property. Similar to what you suggested with the empty array..

Reply:
I guess it works just because we have less strict typescript setup in this package...